test/AngularSpec.js

```diff
@@ -1,7 +1,16 @@
 'use strict';
 
+// Lots of typed array globals are used in this file and ESLint is
+// not smart enough to understand the `typeof !== 'undefined'` guards.
+/* eslint-env es6 */
```

Comment:
This is a little too aggresive imo. Not terribly important though (since this is just a spec file).